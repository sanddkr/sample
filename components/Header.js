import React, { Component } from 'react';
import { Link } from 'react-router-dom';
export class Header extends Component {
  constructor(props) {
    super(props);
    this.state= {};
  }
  render() {
    return (
        <div className="wrapper headerwrap fixed-top">
           <div className="container">
            <header className="text-center">Activate Your Device</header>
          </div>
        </div>
    )
  }
}