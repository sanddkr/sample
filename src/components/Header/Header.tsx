import * as React from 'react';
// import { Link } from 'react-router-dom';
export const Header = (props) => {
  return(
    <div className="wrapper headerwrap fixed-top">
      <div className="container">
        <header className="text-center">Activate Your Device</header>
      </div>
    </div>
  )
};