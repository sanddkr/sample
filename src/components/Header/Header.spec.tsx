import * as React from "react";
import {mount, shallow} from "enzyme";
import * as renderer from 'react-test-renderer';
import Header from './';

describe("Header Component", () => {
    let wrapper, mountwrap;

    beforeEach(() => {
        wrapper = shallow(<Header />);
        mountwrap = mount(<Header />);
    });

    it('Header  component renders correctly', () => {
        expect(wrapper.children().length).not.toBeLessThan(1);
    });

    it('Header  component renders correctly using snapshots', () => {
        const rendered = renderer.create(
            <Header />
        );
        expect(rendered.toJSON()).toMatchSnapshot();
    });
});