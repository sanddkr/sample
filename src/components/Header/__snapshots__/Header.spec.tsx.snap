// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component Header  component renders correctly using snapshots 1`] = `
<div
  className="wrapper headerwrap fixed-top"
>
  <div
    className="container"
  >
    <header
      className="text-center"
    >
      Activate Your Device
    </header>
  </div>
</div>
`;
